app/models/customer_account.rb

```diff
@@ -76,5 +77,9 @@ def self.reset_password(attributes)
     def orders
       ::FlexCommerce::Order.where(customer_account_id: id)
     end
+
+    def create_note(contents:)
+      ::FlexCommerce::Note.create(contents: contents, attached_to_id: self.id, attached_to_type: self.class.name.demodulize)
```

Comment:
I think we should pass any attributes, e.g:

```ruby
def create_note(attributes = {})
  ::FlexCommerce::Note.create(attributes.merge(attached_to_id: self.id, attached_to_type: self.class.name.demodulize))
end
```

This way if we extend the implementation in future, we won't need to update the gem too.